Cargo.toml

```diff
@@ -6,9 +6,9 @@ license = "MIT OR Apache-2.0"
 
 [dependencies]
 # revm
-revm = { git = "https://github.com/scroll-tech/revm", branch = "feat/reth-v78", default-features = false, features = ["enable_eip7702", "enable_eip7623"] }
-revm-primitives = { git = "https://github.com/scroll-tech/revm", branch = "feat/reth-v78", default-features = false }
-revm-inspector = { git = "https://github.com/scroll-tech/revm", branch = "feat/reth-v78", default-features = false }
+revm = { git = "https://github.com/scroll-tech/revm", branch = "scroll", default-features = false, features = ["enable_eip7702", "enable_eip7623"] }
+revm-primitives = { git = "https://github.com/scroll-tech/revm", branch = "scroll", default-features = false }
+revm-inspector = { git = "https://github.com/scroll-tech/revm", branch = "scroll", default-features = false }
```

Comment:
`scroll` is the default branch so we shouldn't have to specify the `branch = ` argument.